prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="index.html">Click here to go to the home page
```

Comment:
After you move your `index.html` to outside of the `contact-us` folder, you'll want to revise your link here a bit.

```html
<a href="../index.html">Click here to go to the home page</a>
```